### PR TITLE
protected method in the controller

### PR DESCRIPTION
**Pull request by zagovorichev**

### CODE REVIEW

File: actions/class.CommonModule.php [hunk 1]
```diff
@@ -357,4 +361,23 @@ protected function getServiceManager()
...
+    {
+        return $this->getOriginalServiceLocator();
+    }
+
+    /**
+     * @param \Zend\ServiceManager\ServiceLocatorInterface $serviceLocator
+     * @return mixed
+     * @security("hide");
+     */
+    public function setServiceLocator(\Zend\ServiceManager\ServiceLocatorInterface $serviceLocator)
```
Comment (martijn-tao) [inline actions/class.CommonModule.php]: Please import this class to prevent the usage of FQNs

**Comment by zagovorichev** [comment]
@martijn-tao Update

**Comment by zagovorichev** [comment]
@martijn-tao Update

File: actions/class.Import.php [hunk 2]
```diff
@@ -42,10 +42,14 @@ class tao_actions_Import extends tao_actions_CommonModule
      */
     private $availableHandlers = [];
 
-    use EventManagerAwareTrait;
     use TaskLogActionTrait;
     use OntologyAwareTrait;
-    
+
+    protected function getEventManager()
```
Comment (martijn-tao) [inline actions/class.Import.php]: Please add a docblock to this method

File: actions/class.PropertiesAuthoring.php [hunk 3]
```diff
@@ -34,10 +34,14 @@
  */
 class tao_actions_PropertiesAuthoring extends tao_actions_CommonModule
 {
-    use EventManagerAwareTrait;
     use OntologyAwareTrait;
     use LoggerAwareTrait;
 
+    protected function getEventManager()
```
Comment (martijn-tao) [inline actions/class.PropertiesAuthoring.php]: Please add a docblock to this method

File: actions/class.Users.php [hunk 4]
```diff
@@ -44,10 +44,14 @@
  */
 class tao_actions_Users extends tao_actions_CommonModule
 {
-    use EventManagerAwareTrait;
     use OntologyAwareTrait;
     use LoggerAwareTrait;
 
+    protected function getEventManager()
```
Comment (martijn-tao) [inline actions/class.Users.php]: Please add a docblock to this method

File: models/classes/actionQueue/implementation/InstantActionQueue.php [hunk 5]
```diff
@@ -39,10 +39,14 @@
 class InstantActionQueue extends ConfigurableService implements ActionQueue
 {
 
-    use EventManagerAwareTrait;
-
     const QUEUE_TREND = 'queue_trend';
 
+    protected function getEventManager()
```
Comment (martijn-tao) [inline models/classes/actionQueue/implementation/InstantActionQueue.php]: Please add a docblock to this method

File: models/classes/export/class.RdfExporter.php [hunk 6]
```diff
@@ -31,7 +32,11 @@
  */
 class tao_models_classes_export_RdfExporter implements tao_models_classes_export_ExportHandler
 {
-    use EventManagerAwareTrait;
+
+    protected function getEventManager()
```
Comment (martijn-tao) [inline models/classes/export/class.RdfExporter.php]: Please add a docblock to this method

**Pull request merged**